### DISCUSSION TITLE
Chrome Extension

### DISCUSSION THREAD

**Discussion by simsra** [twentyhq/twenty#8465] [Ideas]
We're looking to use Twenty and would like a chrome extension to be build to pull data from other source such as Linkedin, Twitter, GitHub etc to create a prospect lead seamlessly instead of manually creating a new lead.

**Comment by BOHEUS** [comment]
@simsra there is a chrome extension for Twenty but it's currently not maintained by the community

**Comment by JhumanJ** [comment]
Hey, I created a small Linkedin extension for Twenty (compatible with self hosted instances). It's not on the chrome store but you can check it out and try it here if you're interested: https://github.com/JhumanJ/twenty-crm-extension. It's great for 1 click Linkedin capture.
<img width="1008" height="642" alt="image" src="https://github.com/user-attachments/assets/8f6cbd48-4abc-48b9-8b1d-225d537c2dd9" />

**Comment by BOHEUS** [comment]
Related to #5264
